leejinwoo/src/components/Comment.tsx

```diff
@@ -0,0 +1,83 @@
+import { ChangeEvent, FormEvent, useState } from "react";
+import Image from "next/image";
+
+// 댓글에 필요한 index(아이디):number / 댓글: string 
+interface IComment{
+    id: number;
+    content: string;
+}
+
+// 댓글 기능 및 좋아요
+function Comment(){
+    // 올라온 댓글 상태관리 [초기에 올라온거, 추가로 올라온거] = IComment에 넣을거임~
+    const [comment, setComment] = useState<IComment[]>([]);
+
+    // 새로올릴 댓글 관리 [추가할댓글 창, 댓글창에 입력되면] = 입력창""이걸로 바꿔주셈
+    const [ncmt, setNcmt] = useState<string>("");
+
+    // 좋아요 상태 관리 [빈 하트, 빈하트가 눌리면] = 참 거짓으로 바꿔주셈~
+    const [heart, setHeart] = useState(true);
```

Comment:
```suggestion
    const [heart, setHeart] = useState<boolean>(true);
```

타입이 추론되겠지만, 이렇게 제너릭을 사용할 수 있는 점 알아주세요~

추가적으로 이름에서 `isLike` 처럼 boolean 형을 유추할 수 있는 이름은 어떨까요?!

Reply:
아하 참고하겠읍니다!